_posts/2022-03-07-gsoc2022.md

```diff
@@ -0,0 +1,126 @@
+from students.
+
+The MDAnalysis community values diversity and is committed to
+providing a productive, harassment-free to every member. Our [Code of
+Conduct] explains the values that we as a community uphold. *Every
+member (and every GSoC student) agrees to follow the [Code of
+Conduct]*.
+
+As a start to get familiar with MDAnalysis and open source development you
+should follow these steps:
+
+## Complete the Quick Start Guide
+
+We have a [Quick Start Guide] explaining the basics of MDAnalysis. You
+should go through it at least once to understand how MDAnalysis is
+used. Continue reading the [User Guide] to learn more.
+
+## Introduce yourself to us
+
+Introduce yourself on the [mailing list]. Tell us what you plan to work
+on during the summer or what you have already done with MDAnalysis.
+
+## Close an issue of MDAnalysis
+
+You must have *at least one commit in the development branch of MDAnalysis* in
+order to be eligible, i.e. you must demonstrate that you have been seriously
+engaged with the MDAnalysis project.  We have a list of [easy bugs][] and
+suggested [GSOC Starter issues][GSOC Starter] to work on in our issue tracker
+on GitHub. *We only accept one [GSOC Starter issue][GSOC Starter] per
+applicant* so that everybody gets a chance. If you want to dive deeper, tackle
+some of the other issues.
+
+We also appreciate if you write more tests or update/improve our documentation.
+
+We recommend you start your application by working on an issue. It will give
+you a better understanding of MDAnalysis as a project and improve the quality
+of your application.
+
+To start developing for MDAnalysis have a look at our
+guide on [contributing to MDAnalysis][dev-guide] and write to us on
+the [mailing list][] if you have more questions about setting up a
+development environment or how to contribute.
+
+
+— Hugo @hmacdope, Micaela @micaela-matta, Jonathan @jbarnoud, Richard @richardjgowers, Lily @lilyminium, Fiona @fiona-naughton, Irfan @IAlibay, Oliver @orbeckst
+
+
+[org]: https://summerofcode.withgoogle.com/organizations/
+[Apply]: https://summerofcode.withgoogle.com/student-signup/
+[Code of Conduct]: {{ site.baseurl }}/pages/conduct/
+[eligibility criteria]: https://developers.google.com/open-source/gsoc/faq#what_are_the_eligibility_requirements_for_participation
+[own requirements]: {{ site.github.wiki }}/Google-Summer-Of-Code#our-expectations-from-students
+[easy bugs]: https://github.com/MDAnalysis/mdanalysis/issues?q=is%3Aopen+is%3Aissue+label%3ADifficulty-easy
+[GSOC Starter]: https://github.com/MDAnalysis/mdanalysis/issues?q=is%3Aopen+is%3Aissue+gsoc+label%3A%22GSOC+Starter%22
+[Quick Start Guide]: https://userguide.mdanalysis.org/stable/quickstart.html
+[User Guide]: https://userguide.mdanalysis.org/
+[ideas]: {{ site.github.wiki }}/GSoC-2022-Project-Ideas
+[gsoc]: https://summerofcode.withgoogle.com/
+[dev-guide]: https://www.mdanalysis.org/UserGuide/contributing.html
```

Comment:
```suggestion
[dev-guide]: https://userguide.mdanalysis.org/stable/contributing.html
```

redirects to the same thing, but for whomever copies this next year in case the redirect one day goes away.